### PR TITLE
output file cleanup

### PR DESCRIPTION
**Pull request by funnell**

### CODE REVIEW

File: workflow/rules/preprocess.smk [hunk 1]
```diff
@@ -322,43 +322,6 @@ rule merge_fastq_pair:
         """
 
 
-rule merge_primary_host_align_bams:
```
Comment (nickp60) [inline workflow/rules/preprocess.smk:325]: Why is this moved? Not a big deal, just makes diffing more difficult

File: workflow/rules/preprocess.smk [hunk 2]
```diff
@@ -479,35 +458,54 @@ rule merge_logs_for_multiqc:
...
+    """
+    input:
+        bams=expand(
+            "01-bowtie/{sample}_shard{shard}.{db}.bam",
+            sample=config["sample"],
+            shard=SHARDS,
+            db=bowtie2_human_db_name,
+        ),
+    output:
+        bam=temp(f"{{sample}}.{bowtie2_human_db_name}.bam"),
```
Comment (nickp60) [inline workflow/rules/preprocess.smk:475]: I think these shouldn't be temp'ed.  Confusion probably arose cause I forgot to add those to the rule all cause it was being forced by the xhla rule.  I think we had discussed wanting to retain the primary human alignment?

File: workflow/rules/preprocess.smk (hunk 1 above)
Comment (funnell) [reply workflow/rules/preprocess.smk:325]: Yes sorry for adding difficulty! The output was only being used by xHLA so I moved it closer to that rule.

File: workflow/rules/preprocess.smk (hunk 2 above)
Comment (funnell) [reply workflow/rules/preprocess.smk:475]: Ah I remember that conversation now! Thanks for reminding me. We decided to retain bzipped fastq files for human/mouse reads. The problem with the bam file is it's an amalgamation of alignments from two different aligners!

Comment (nickp60) [reply workflow/rules/preprocess.smk:475]: The bam created by this rule is just the primary human alignment from bowtie; the merging is just combining across shards

Comment (nickp60) [reply workflow/rules/preprocess.smk:475]: I think we decided to keep both the primary human bam and the the _fastqs_ from any aligner to either mouse or human

Comment (funnell) [reply workflow/rules/preprocess.smk:475]: Oh I see! I misunderstood. I'll untemp and put the bam somewhere appropriate...

Comment (funnell) [reply workflow/rules/preprocess.smk]: ok, I've updated and just put the (un-temped) bams into the "host/" directory along with the all host fastq files

Comment (nickp60) [reply workflow/rules/preprocess.smk]: awesome
